src/Mapper/MediaMapper.php

```diff
@@ -25,6 +43,13 @@ function (Media $media) {
         );
     }
 
+    /**
+     * Summary of getTrickGroupModel
+     *
+     * @param Media $media Media
+     *
+     * @return HomeMedia
+     */
     public function getMediaModel(Media $media): HomeMedia
```

Comment:
```suggestion
    public function getMediaModel(Media $mediaEntity): HomeMedia
```

Je pense qu'en sufffixant c'est plus lisible. Sinon on doit regardé d'où vient la classe pour savoir si c'est une entity ou un model. Mais ce n'est qu'un avis.

Reply:
En effet ok je m'en occupe ! Merci